enterprise_access/apps/content_assignments/models.py

```diff
@@ -142,6 +142,14 @@ class Meta:
             "ContentMetadata.content_key in enterprise-catalog."
         ),
     )
+    content_title = models.CharField(
```

Comment:
later on we can develop a process that updates `content_title` and `learner_email` from authoritative sources if need be